api/src/opentrons/protocol_api/core/engine/protocol.py

```diff
@@ -140,11 +140,20 @@ def load_labware(
         else:
             module_location = DeckSlotLocation(slotName=location)
 
+        custom_labware_params = (
+            self._engine_client.state.labware.find_custom_labware_load_params(
+                load_name, namespace, version
+            )
+        )
+        namespace, version = engine_param_validation.resolve_load_labware_params(
+            custom_labware_params, namespace, version
+        )
```

Comment:
(Comment 3 of 3) I think the core logic pipeline is good, we should just shuffle some stuff around:

```suggestion
        custom_labware_params = (
            self._engine_client.state.labware.get_all_custom_labware_load_params()
        )
        namespace, version = engine_param_validation.resolve_load_labware_params(
            load_name=load_name,
            namespace=namespace,
            version=version,
            custom_labware_params=custom_labware_params,
        )
```